.gitignore

```diff
@@ -86,3 +86,6 @@ typings/
 # JSDoc output
 out/
 docs/
+
+# TypeScript build output
+dist/
```

Comment:
Interesting. Yeah I wasn't quite sure how all that is supposed to work. Is the .ts source what gets bundled? Is the .js output? Then again, I don't know if this will ever actually end up getting bundled as an NPM package, since this is an application rather than a library. But maybe I'm just bringing my biases over from the Python world. :-D